.changeset/odd-otters-tan.md

```
@@ -0,0 +1,5 @@
+---
+"@tm/cli": minor
+---
+
+testing this stuff out to see how the release candidate works with monorepo
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)